client/src/telethon/_impl/tl/core/reader.py

```diff
@@ -74,7 +76,7 @@ def read_bytes(self) -> memoryview:
 
     _get_ty = staticmethod(_bootstrap_get_ty)
 
-    def read_serializable(self, cls: Type[T]) -> T:
+    def read_serializable(self, cls: Type["Serializable"]) -> "Serializable":
         # Calls to this method likely need to ignore "type-abstract".
```

Comment:
Is this fixed with your change? If so, the comment can be removed.

Reply:
I forgot to check through mypy